packages/web/components/TechnologyForm/MapAndAttachments/styles.js

```diff
@@ -92,17 +92,88 @@ export const Media = styled.img`
 `;
 
 export const IconRow = styled.div`
-	display: flex;
-	flex-direction: ${({ row }) => (row ? 'row' : 'column')};
-	align-items: center;
+	${({ row, alignItems }) => css`
+		display: flex;
+		flex-direction: ${row ? 'row' : 'column'};
 
-	& > * {
-		margin-bottom: 1rem;
-	}
+		& > * {
+			margin-bottom: 1rem;
+		}
+
+		& > a {
+			padding: 0;
+			margin-right: 1rem;
+			display: flex;
+		}
+
+		align-items: ${alignItems || 'center'};
+	`}
+`;
+
+export const Button = styled.button.attrs(() => ({ type: 'button' }))`
+	${({ theme: { colors } }) => css`
+		border: none;
+		outline: none;
 
-	& > a {
-		padding: 0;
-		margin-right: 1rem;
 		display: flex;
-	}
+		align-items: center;
+
+		padding: 0.2rem;
+		padding-left: 0.6rem !important;
+		padding-right: 0.6rem !important;
+		line-height: 2.4rem;
+
+		text-transform: uppercase;
+		font-size: 1.4rem;
+		font-weight: 700;
+		color: ${colors.red};
+		border: 2px solid ${colors.red};
+
+		> svg {
+			margin-right: 0.4rem;
+		}
+
+		:hover,
+		:focus {
+			color: ${colors.white};
+			background: ${colors.red};
+		}
+	`}
+`;
+
+export const RadioWrapper = styled.div`
+	${({ theme: { colors } }) => css`
+		display: flex;
+		align-items: center;
+		white-space: nowrap;
+
+		> input {
+			appearance: none;
+		}
+		> input:checked + label {
+			color: ${colors.white};
+			background-color: ${colors.secondary};
+			border: 2px solid ${colors.secondary};
+		}
```

Comment:
If you wrapp the input with the label tag, as suggested above, you'll need to update the style below:
```suggestion
		input {
			appearance: none;
		}
		input:checked + label {
			color: ${colors.white};
			background-color: ${colors.secondary};
			border: 2px solid ${colors.secondary};
		}
```

Reply:
Hey @lcnogueira, this `input:checked + label` won't work if I wrap input with label. Any thoughts?

Reply:
I could not achieve a good result by wrapping input 😞

Reply:
We have to style the whole label to fill it's background, otherwise it'll look like this:

![image](https://user-images.githubusercontent.com/13091635/97297174-67913b00-1830-11eb-80dc-64f3860c91fb.png)

Reply:
Any thoughts @calaca ?

Reply:
@luizeboli not really. I guess we should keep it as you did before, then.

I do have a visual suggestion:
![image](https://user-images.githubusercontent.com/10481028/97298316-079b9400-1832-11eb-94c7-81013562b45a.png)
Don't you think it looks better with all buttons with the same width as shown above the first picture?

Reply:
@calaca At first it looks slightly better, but on bigger breakpoints maybe it looks weird:

![image](https://user-images.githubusercontent.com/13091635/97302821-53e9d280-1838-11eb-8402-fadb87598df8.png)

Reply:
I'll close this, ok? We can discuss it later as it's of such importance that @JoaoPauloSMoura also participate in these kind of discussions.

Reply:
Agreed 👍